backend/substrapp/urls.py

```diff
@@ -27,6 +27,9 @@
 router.register(r'testtuple', TestTupleViewSet, base_name='testtuple')
 router.register(r'task', TaskViewSet, base_name='task')
 router.register(r'compute_plan', ComputePlanViewSet, base_name='compute_plan')
+router.register(r'compositetraintuple', CompositeTraintupleViewSet, base_name='compositetraintuple')
+router.register(r'compositealgo', CompositeAlgoViewSet, base_name='compositealgo')
+router.register(r'compositealgo', CompositeAlgoPermissionViewSet, base_name='compositealgo')
```

Comment:
Why not use underscores in the routes themselves? We already have them for data_manager, data_sample and compute_plan.